Readme.md

```diff
@@ -84,44 +84,63 @@ the specifics for *pino-multi-stream*:
 
 [pinoapi]: https://github.com/pinojs/pino#api
 
-<a id="caveats"></a>
-## Caveats
+### pinoms.multistream(streams)
+
+Manually create a single `multistream` as used internally by the
+wrapper:
+
+```js
+var fs = require('fs')
+var pino = require('pino')
+var multistream = require('pino-multi-stream').multistream
+var streams = [
+  {stream: fs.createWriteStream('/tmp/info.stream.out')},
```

Comment:
@davidmarkclements this should clarify this: https://github.com/pinojs/pino-multi-stream/blob/a30efc71f6445f5f5bfd6f267378edde0dabbb9b/test/multistream.js#L8-L28